README.md

```diff
@@ -120,6 +120,16 @@ jlpm run build
 jupyter lab build
 ```
 
+Or start jupyter in watch mode:
+```bash
+jupyter-lab --watch
```

Comment:
```suggestion
jupyter lab --watch
```

This is the recommended syntax.

Reply:
My bad, old habits. It was needed before this:
https://github.com/jupyter/jupyter_core/pull/148/files#diff-043250408cc76a63bbd3f7cd15e50cdbR122

Reply:
No problem thanks for contributing 😉